model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/PersistentUserSessionProvider.java

```diff
@@ -1030,8 +1031,18 @@ public void migrateNonPersistentSessionsToPersistentSessions() {
         AtomicInteger currentBatch = new AtomicInteger(0);
         var persistence = ComponentRegistry.componentOf(sessionCache, PersistenceManager.class);
         if (persistence != null && !persistence.getStoresAsString().isEmpty()) {
+            ArrayList<Throwable> errors = new ArrayList<>();
```

Comment:
you don't need an array list since it will only have, at most, a single exception.

Reply:
What Java container do you suggest? I need something that's valid within a lambda...

Reply:
we have `org.infinispan.commons.util.ByRef` in Infinispan for this type of case :)

Reply:
ok, let's use that. PR updated.